### PR TITLE
Add link to Google Groups

### PR DESCRIPTION
**Pull request by aniabui**
I added a link to Google Groups in 'Find us' section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,5 +112,6 @@ This software is released under the terms of the MIT License. See the
 
 ## Find us
 
-Jangouts developers can be usually found at
-[#jangouts IRC channel at Freenode](irc://irc.freenode.net/jangouts)
+Jangouts developers can be usually found at:
+ - [#jangouts IRC channel at Freenode](https://webchat.freenode.net/?channels=%23jangouts)
+ - [Google Groups](https://groups.google.com/forum/#!forum/jangouts)
```
Comment (imobachgs) [inline README.md]: Just a minor comment: I would say 'Google Group' (singular) or 'Mailing List' as you prefer.

Comment (ancorgs) [inline README.md]: Well, if we are going to look that close, :wink: I would even state that IRC is good to reach the developers at a given point in time and the Google group is more targeted for general discussions within the whole community. However you want to word that.

Comment (imobachgs) [inline README.md]: That's maybe too much :) Just changing it to *singular* is enough for me :)

Comment (aniabui) [inline README.md]: I just want to point out that the it's officially called Google GroupS, not Google Group :)

Comment (imobachgs) [inline README.md]: Yes, but Jangouts group is only one :) Anyway, let's keep it as it is :)

Thanks!

**Pull request merged**